app/components/provider_interface/application_timeline_component.rb

```diff
@@ -87,10 +87,32 @@ def change_offer_events
       end
     end
 
+    def interview_events
+      @activity_log_events.select { |e| e.audit.auditable_type == 'Interview' }.map do |event|
+        Event.new(
+          interview_title_for(event.audit),
+          actor_for(event),
+          event.created_at,
+          'View interview',
+          provider_interface_application_choice_path(application_choice),
```

Comment:
Here we should only link to the interview conditionally. As I'm utilising the discard gem you should be able to check if the interview has been cancelled by using `interview.discarded?`. In that instance there shouldn't be a link.

We can update the path to link to the interviews path later as that code is stil waiting to be reviewed